src/Linter/Sniff/IndentationSniff.php

```diff
@@ -105,6 +108,14 @@ public function sniff(array $tokens, File $file, LinterConfiguration $configurat
      */
     private function isEmptyLine(array $tokensInLine): bool
     {
+        $tokensInLine = array_values(array_filter($tokensInLine, function (TokenInterface $t): bool {
+            return $t->getType() !== TokenInterface::TYPE_EMPTY_LINE;
+        }));
+
+        if (count($tokensInLine) === 0) {
+            return true;
+        }
+
```

Comment:
shouldn't there be just a return false? Can `$tokensInLine` also be exactly 1? In this case the method doesn't return anything instead of a boolean value and than it crashes.